server/models/index.js

```diff
@@ -1,2 +1,18 @@
 // Import models here
-// schemas here and relationships
\ No newline at end of file
+// schemas here and relationships
+const User = require('./User')
+const Character = require('./Character')
+const Message = require('./Message')
+
+User.hasMany(Character)
+User.hasMany(Message) 
+// sender
+User.hasMany(Message)
+// receiver
+Character.hasMany(Message)
+// sender
+Character.hasMany(Message)
```

Comment:
looking at this I realized you have User.hasMany(Message) AND Message.belongsToMany(User)  --these two methods do different things and cannot be used together. 
Here is some more info about it and we can talk some more about it today or tomorrow :) 
https://gist.github.com/elliette/20ddc4e827efd9d62bc98752e7a62610#:~:text=You%20should%20not%20use%20use,you%20should%20never%20write%20both.

Reply:
after reviewing the code from your previous project I have found that this set up --https://github.com/branjames117/devspark/blob/main/models/index.js
--is designed for one on one message but not group messaging.

We will have to do some research on Group Messaging Database set ups or move group messaging into Future Dev.